src/routes.js

```diff
@@ -1,23 +1,21 @@
 import { Route, Switch, Redirect } from 'react-router-dom';
 import PropTypes from 'prop-types';
 import React, { lazy, Suspense } from 'react';
-import { AppPlaceholder } from './presentational-components/shared/loader-place-holders';
+import { AppPlaceholder } from './presentational-components/shared/loader-placeholders';
 
 const Groups = lazy(() => import('./smart-components/group/groups'));
-const Group = lazy(() => import('./smart-components/group/group'));
 
 const paths = {
   rbac: '/',
-  groups: '/groups',
-  group: '/group/:id'
+  groups: '/groups'
 };
 
 const InsightsRoute = ({ rootClass, ...rest }) => {
```

Comment:
Just FYI, the classNames ent do nothing. Insights route can be removed entirely. This: `page__${rootClass}, 'pf-l-page__main', 'pf-c-page__main'` shound be set in index.html.

Reply:
sounds good - I will have these changes in a separate PR as these do not apply to this PR, that is already too big